script.js

```diff
@@ -22,3 +22,134 @@ function mobileMenu() {
+  if (index % 2 === 1) {
+    container.className = 'card flex flex-column bg row-reverse';
+  } else {
+    container.className = 'card flex flex-column bg';
+  }
+  container.innerHTML = ` <img src="${ele.img}" alt="project photo">
+    <div class="content">
+      <h3 class="heading bold cap">${ele.name}</h3>
+      <ul class="info flex">
+        <li class="category flex cross-center ellipse line-height highlight uppe">CANOPY</li>
+        <li class="category flex cross-center ellipse line-height highlight uppe">Back End Dev</li>
+        <li class="category flex cross-center ellipse line-height highlight uppe">2015</li>
+      </ul>
+      <p>
+      ${ele.description}
+      </p>
+      <ul class="tags flex">
+        <li class="line-height round-borders">${ele.technologies[0]}</li>
+        <li class="line-height round-borders">${ele.technologies[1]}</li>
+        <li class="line-height round-borders">${ele.technologies[2]}</li>
+      </ul>
+      <button class="btn bg round-borders cap " type="button">see project</button>
+    </div>`;
+  return container;
+});
+
+const works = document.querySelector('#pop-works');
+for (let i = 0; i < ProjectElments.length; i += 1) {
+  works.appendChild(ProjectElments[i]);
+}
+
+const ProjectPopUp = (event) => {
+  const cardTitle = event.target.parentNode.children[0].innerHTML;
+  let objItem;
+  for (let i = 0; i < cardPopUp.length; i += 1) {
+    if (cardPopUp[i].name === cardTitle) {
+      objItem = cardPopUp[i];
+    }
+  }
+
+  const container = document.createElement('article');
+  container.className = 'card-pop card flex flex-column bg';
+  container.setAttribute('id', 'card-popUp');
+
+  container.innerHTML = `
+      <div class="first">
+      <img id="exit-popUp" class="exit-card" src="images/exit2.svg" alt="exit">
+    <h3 class="heading bold cap">${objItem.name}</h3>
+    </div>
+    <ul class="info flex">
+      <li class="category flex cross-center ellipse line-height highlight uppe">CANOPY</li>
+      <li class="category flex cross-center ellipse line-height highlight uppe">Back End Dev</li>
+      <li class="category flex cross-center ellipse line-height highlight uppe">2015</li>
+    </ul>
+        <img class="pro-img" src="${objItem.img}" alt="project photo">
+        <div class="content-pop">
+          <p>
+          ${objItem.description}
+          </p>
+          <div class="lang-btn">
+          <ul class="popUp-tags tags flex">
+            <li class="line-height round-borders">${objItem.technologies[0]}</li>
+            <li class="line-height round-borders">${objItem.technologies[1]}</li>
+            <li class="line-height round-borders">${objItem.technologies[2]}</li>
+          </ul>
+          <div class="btn2">
+            <button class="b1 bg round-borders cap " type="button">see live <img src="images/arrowCir.svg" alt="goto"></button>
+            <button class="b2 bg round-borders cap " type="button">see Source <img src="images/cat.svg" alt="sourceCode"></button>
```

Comment:
- [ ] [OPTIONAL] On hover the social media icon is disappearing,

![image](https://user-images.githubusercontent.com/77038610/146387755-f3f65a19-06ee-45a3-a875-4201bc1c4c79.png) 

I recommend you to set the color of the icon to white on hover 😉